docs/README.md

```diff
@@ -54,14 +56,26 @@ applications, entire subnets, and everything in between.
 </p>
 -->
 
+## Getting Started
+
+### Option 1: Cloud (Recommended)
```

Comment:
There's no `Option 2` from what I saw.  Was there supposed to be?  If not, should we remove this header?